src/mainPage/footer.ts

```diff
@@ -0,0 +1,19 @@
+import { IndexedFormula } from 'rdflib'
+import { initFooter } from 'solid-ui'
```

Comment:
question: thinking those GitHub action errors will go away once solid-ui change is in main...

Reply:
yes correct. We have them because this change depends on solid-ui changes to be merged first.